api/kubernetes/cli/deploment.go

```diff
@@ -0,0 +1,26 @@
+package cli
+
+import (
+	"context"
+
+	portainer "github.com/portainer/portainer/api"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GetDeployments will get deployments within a given namespace.
+// If the parameter 'namespace' is an empty string, all deployments in the kubernetes env will be retrived.
+func (kcl *KubeClient) GetDeployments(namespace string) ([]portainer.KubernetesDeployment, error) {
```

Comment:
Query by label is a good suggestion. To make it simple, I changed function name to IsUniqueStackName. Because I don't want the stacks, I just want to know if there are stacks with this name.
The code is passed by checkers. Please review again.